share/instruments/instruments.xml

```diff
@@ -5371,7 +5371,7 @@
                   <bracketSpan>2</bracketSpan>
                   <barlineSpan>2</barlineSpan>
                   <clef staff="2">F</clef>
-                  <barlineSpan>1</barlineSpan>
+                  <barlineSpan staff="3">1</barlineSpan>
```

Comment:
3 staves on handbells?

Reply:
This happened because the old instruments.xml has multiple `<barlineSpan>` tags for this instrument, so I will need to intervene manually in this case.

Reply:
There's `<barlineSpan>2</barlineSpan>` and `<barlineSpan>1</barlineSpan>`, that indeed doesn't look right